src/Api/AdminConsole/Models/Response/Organizations/OrganizationUserResponseModel.cs

```diff
@@ -127,13 +134,25 @@ public OrganizationUserUserDetailsResponseModel(OrganizationUserUserDetails orga
         Groups = organizationUser.Groups;
         // Prevent reset password when using key connector.
         ResetPasswordEnrolled = ResetPasswordEnrolled && !organizationUser.UsesKeyConnector;
+        ManagedByOrganization = managedByOrganization;
     }
 
     public string Name { get; set; }
     public string Email { get; set; }
     public string AvatarColor { get; set; }
     public bool TwoFactorEnabled { get; set; }
     public bool SsoBound { get; set; }
+    /// <summary>
+    /// Indicates if the organization manages the user.
+    /// </summary>
+    /// <remarks>
+    /// An organization manages a user if the user's email domain is verified by the organization and the user is a member of it.
+    /// The organization must be enabled and able to have verified domains.
```

Comment:
I know I said to add xmldoc comments, but there's also a risk in putting very specific comments everywhere - they're more likely to become out of date if the logic changes.

I suggest keeping this a bit more general, e.g. "Indicates if the organization manages the user. If a user is "managed" by an organization, the organization has greater control over their account, and some user actions are restricted."